assets/src/components/customers/CustomersPage.tsx

```diff
@@ -156,14 +179,28 @@ class CustomersPage extends React.Component<Props, State> {
             />
           </Box>
 
-          <Box mb={3}>
+          <Flex mb={3} sx={{justifyContent: 'space-between'}}>
             <Input.Search
               placeholder="Search customers..."
               allowClear
               onSearch={this.handleSearchCustomers}
               style={{width: 400}}
             />
-          </Box>
+
+            <Button
+              type="primary"
+              icon={<PlusOutlined />}
+              onClick={this.handleCreateNewCustomer}
+            >
+              New customer
+            </Button>
+          </Flex>
+
+          <NewCustomerModal
+            visible={isModalOpen}
+            onCancel={this.handleCreateCustomerCancelled}
+            onSuccess={this.handleCreateCustomerSuccess}
+          />
```

Comment:
Nit: probably not necessary for this diff, but it might be worthwhile to consider consolidating button + modal logic into a separate component so that the current component (i.e. CustomersPage) doesn't have to also hold all of this logic related to whether the modal is open (see https://github.com/papercups-io/papercups/pull/718).

Reply:
good idea :+1:

Reply:
done!